### ISSUE TITLE
Check password after second factor code entered

### ISSUE THREAD

**Issue by floreal797** [nextcloud/twofactor_totp#184]
Current behavior:
If we enter a wrong password, the error is before we enter the second factor code.

Expected behavior:
For a better security, it is better to say if password is correct or not only after the second factor entered (without saying which is wrong).

**Comment by ChristophWurst** [comment]
I can't say if it would make a lot of sense to change it this way, but in any case this work flow would require changes in the nextcloud server component. Hence it's better to open a new issue there.

cc @LukasReschke 

Personally, I've never seen a platform that would show password verification errors this way.

**Comment by floreal797** [comment]
I moved issue in server component, like you say:
https://github.com/nextcloud/server/issues/5935

**Comment by ChristophWurst** [comment]
Thanks!
